### DISCUSSION TITLE
Extrapolation

### DISCUSSION THREAD

**Discussion by zasdaym** [grafana/clickhouse-datasource#777] [Ideas]
Other ClickHouse plugin supports extrapolation:

- Description: https://github.com/Altinity/clickhouse-grafana?tab=readme-ov-file#faq
- Implementation: https://github.com/Altinity/clickhouse-grafana/blob/master/src/datasource/sql_series.ts#L184

It will be great if the official plugin have this feature too.

**Comment by SpencerTorres** [comment]
Do you have an example screenshot of what this looks like enabled/disabled? I think I can picture it and why people would want it, but is this not possible with Grafana's transformations? Personally, I would be cautious to add estimated data to a panel that wasn't directly read from the database.
